com.civicrm.hrjobroles/js/src/job-roles/controllers/job-role-panel.html

```diff
@@ -326,7 +326,18 @@
               <table class="table">
                 <thead>
                   <tr>
-                    <th style="width:40%">Funder</th>
+                    <th style="width:40%">
```

Comment:
It seems that we have identical markup in 2 files. Do you think it may be a good idea to extract repeating code into a separate `html` partial file?

Reply:
I will suggest we leave as is because then we will to start taking of additional logic particular to each. For instance when `Add New Funder` is clicked, different parameters are passed across which may also make page partial file more complicated 🤷‍♂️

Reply:
Fair enough.